### PR TITLE
:bug: ensure that the data will be always store

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Ensure that the data will be stored always

### CODE REVIEW

File: pkg/plugins/golang/v2/scaffolds/api.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ package scaffolds
 
 import (
 	"fmt"
+	cfgv2 "sigs.k8s.io/kubebuilder/v3/pkg/config/v2"
```
Comment (Adirio) [inline pkg/plugins/golang/v2/scaffolds/api.go:21]: NIT: imports

File: testdata/project-v3-config/PROJECT [hunk 2]
```diff
@@ -40,4 +40,10 @@ resources:
   webhooks:
     defaulting: true
     webhookVersion: v1
+- controller: true
+  domain: testproject.org
+  group: crew
+  kind: Laker
+  path: sigs.k8s.io/kubebuilder/testdata/project-v3-config/api/v1
```
Comment (Adirio) [inline testdata/project-v3-config/PROJECT:47]: `Laker` didn't scaffold an api so this path does not exist, it cannot be persisted to the config file.

File: testdata/project-v3/PROJECT [hunk 3]
```diff
@@ -40,4 +40,10 @@ resources:
   webhooks:
     defaulting: true
     webhookVersion: v1
+- controller: true
+  domain: testproject.org
+  group: crew
+  kind: Laker
+  path: sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1
```
Comment (Adirio) [inline testdata/project-v3/PROJECT:47]: Same as above.

**Review by Adirio — CHANGES_REQUESTED** [review]

**Comment by camilamacedo86** [comment]
close in favor of another PR

**Pull request closed without merging**